components/tidb_query_expr/src/impl_string.rs

```diff
@@ -1118,7 +1131,7 @@ mod tests {
             (Some("TiKV"), Some(4i64)),
             (Some("あなたのことが好きです"), Some(33i64)),
             (Some("분산 데이터베이스"), Some(25i64)),
-            (Some("россия в мире  кубок"), Some(38i64)),
+            (Some("россия в мире  кубок"), Some(38i64)),
```

Comment:
Please don't introduce unrelated changes.

Reply:
OK, I have fixed it.